csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -25,7 +25,7 @@ public class SessionOptions:IDisposable
     {
         public IntPtr _nativePtr;
         protected static readonly Lazy<SessionOptions> _default = new Lazy<SessionOptions>(MakeSessionOptionWithCpuProvider);
-        private static string[] cudaDelayLoadedLibs = { "cublas64_100.dll", "cudnn64_7.dll" };
+        private static string[] cudaDelayLoadedLibs = { "cublas64_91.dll", "cudnn64_7.dll" };
```

Comment:
can this be done with a conditional compiler variable set from CMAKE?

Reply:
We can investigate this in future...probably automate, or obviate the need for this. Currently, it's a safety check, and would be good to include as part of the release.